lib/gc/Transforms/IterativeTilingAndFusion.cpp

```diff
@@ -157,57 +157,46 @@ exactTilingOnPackUnPackFilter(RewriterBase &rewriter,
 
   SmallVector<OpFoldResult> tileSizes = candidate.getMixedSizes();
   // collect target TileSizes and InnerTileSize to compare
-  SmallVector<OpFoldResult> targetTileSizes, targetInnerTileSizes;
+  SmallVector<OpFoldResult> tileSizesOnInnerDims, innerTiles;
```

Comment:
Please change the comment on line 159 accordingly

Reply:
changed :)